OracleOGG/MySQL/README.md

```diff
@@ -0,0 +1,64 @@
+Oracle GoldenGate for Oracle on Docker
+===============
+Sample Docker build files to facilitate installation, configuration, and environment setup for DevOps users. 
+
+For more information about Oracle GoldenGate 12c please see the Oracle GoldenGate Online Documentation at http://docs.oracle.com/goldengate/c1221/gg-winux/index.html.
+
+For more information about Oracle Database 12c, please see the Oracle Database Online Documentaion at http://docs.oracle.com/en/database/index.html.
+
+## How to build Oracle GoldenGate for Oracle on Docker
+This project offers sample Dockerfile and other associated file for this build:
+ * V100691-01.zip (Oracle GoldenGate 12c for MySQL(12.2.0.1.1))
+ * mysql-community-server-minimal-5.7.18-1.el7.x86_64.rpm (MySQL Community Edition 5.7.18)
+ * Dockerfile - Build file for Docker image
+ * entrypoint.sh - steps required to install MySQL 5.7 and Oracle GoldenGate 12c
+
+To rebuild the Oracle GoldenGate for MySQL Docker image, use the following command:
+
+	docker build -t oggmysql:12.2.0.1.1 -f Dockerfile .
+ 
+## Running Oracle GoldenGate for Oracle in a Docker container
+To run your Oracle Database Docker image use the **docker run** command as follows:
+
+	docker run -d --privileged --name oggmysql \

```

Comment:
We cannot require `--privileged` mode. Please rework this image so this is not required.